src/time.ts

```diff
@@ -0,0 +1,9 @@
+import moment from "moment";
+
+export function now() {
+  return moment().unix();
+}
+
+export function relativeMinutesToTimestamp(minutes: number) {
```

Comment:
❓ I don't understand. You pass `alpha_expiry` to it but you label it `minutes`. `alpha_expiry` is in seconds from unix epoch though.

Then yeah, `seconds * 60` = minutes from epoch... and you add it to `now()`? 

Or is `alpha_expiry` not the one returned by comit_node http api.

Reply:
In this case `alpha_expiry` is the one chosen through `SendSwapRequest` form, which is in minutes relative to current time. The function `relativeMinutesToTimestamp` converts it to the absolute timestamp that the comit node expects.